src/components/forms/DatePicker/DatePicker.tsx

```diff
@@ -24,20 +24,21 @@ import {
 } from './utils'
 import { Calendar } from './Calendar'
 
-interface DatePickerProps {
+export interface DatePickerProps {
   id: string
   name: string
   className?: string
   disabled?: boolean
   required?: boolean
   defaultValue?: string
-  minDate: string
+  minDate?: string
```

Comment:
Since this prop has a default value, I made it optional so that it isn't required for the props of either datepicker in the DateRangePicker. I think it was intended to be optional in the initial Datepicker release, given the default prop specified for storybook, and the default value when extracting props.